src/sinks/clickhouse.rs

```diff
@@ -175,7 +175,7 @@ impl HttpSink for ClickhouseConfig {
 
 async fn healthcheck(client: HttpClient, config: ClickhouseConfig) -> crate::Result<()> {
     // TODO: check if table exists?
-    let uri = format!("{}/?query=SELECT%201", config.endpoint);
+    let uri = format!("{}/?query=SELECT%201", config.endpoint.with_default_parts());
```

Comment:
I feel like we want to document what the defaults would be if not provided.

Reply:
Not sure, if we need to do it in this PR, since we already have other sinks, which also do not document this behaviour.

I agree, that we need to write a documentation about it. I just propose to write it in a some generic way for all affected components.